### PR TITLE
add makeAxis support to JSON parsing

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: b56c49a182154c66dd20720825e0ea55e7d3713f available at: https://gist.github.com/10ac56487ba8d6d3b16d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6237.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6237.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8a03383adc237f77105edb984d48a036b82584af available at: https://gist.github.com/10ac56487ba8d6d3b16d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6237.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6237.dmg

File: interface/resources/controllers/keyboardMouse.json [hunk 1]
```diff
@@ -2,17 +2,23 @@
...
-        { "from": ["Keyboard.A", "Keyboard.Left", "Keyboard.TouchpadLeft"],
-          "when": [ "Application.InHMD", "Application.ComfortMode" ],
+        
+        { "from": { "makeAxis" : ["Keyboard.MouseMoveLeft", "Keyboard.MouseMoveRight"] },
+          "when": [ "Application.InHMD", "Application.ComfortMode", "Keyboard.RightMouseClick" ],
           "to": "Actions.StepYaw",
           "filters":
             [
+                "constrainToInteger",
                 { "type": "pulse", "interval": 0.5 },
```
Comment (jherico) [inline interface/resources/controllers/keyboardMouse.json]: an interval of 0.5 for mouse movement is intolerably slow.  I'd recommend no more than 0.2.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 70f31563284c06f7f58c2ce8f2f2794cc392f1bd available at: https://gist.github.com/10ac56487ba8d6d3b16d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6237.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6237.dmg

**Pull request merged**